Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.yml

```diff
@@ -5074,6 +5078,12 @@ script:
       isArray: false
       name: ran_once_flag
       required: false
+    - default: true
+      description: ''
+      isArray: true
+      name: endpoint
+      required: true
```

Comment:
Is this inteneded?

Reply:
Nope
Format did it (will investigate and open a ticket if needed)
removed